lib/lexer.js

```diff
@@ -285,7 +285,7 @@ Lexer.prototype = {
   urlchars: function() {
     var captures;
     if (!this.isURL) return;
-    if (captures = /^[\/:@.;?&=*!,<>#%0-9]+/.exec(this.str)) {
+    if (captures = /^\+[a-z]+/.exec(this.str) ? /^[\/:@.;?&+=*!,<>#%0-9]+/.exec(this.str) : /^[\/:@.;?&=*!,<>#%0-9]+/.exec(this.str)) {
```

Comment:
Can we refact these code to be more clear ? (e.g add comments for Regex
> Thanks for your great contribution, i invite you to mangage stylus'issues and pull requests

Reply:
I'll see what I can do in terms of refactor, regex always looks messy IMO 😄 - but sure, I can definitely add comments!